app/lib/actions/katello/repository/sync.rb

```diff
@@ -16,7 +16,7 @@ class Sync < Actions::EntryAction
         #   of Katello and we just need to finish the rest of the orchestration
         # rubocop:disable MethodLength
         # rubocop:disable CyclomaticComplexity
-        def plan(repo, pulp_sync_task_id = nil, options = {})
+        def plan(repo, _pulp_sync_task_id = nil, options = {})
```

Comment:
can you remove the task_id parameter?  (and remover any reference to it?)

Reply:
actually it looks like you already removed the reference to it in Orchestration :)

Reply:
oh actually sorry, no it'd need to be cleaned up in the controller and tests, yes lets get rid of this argumet :)